seatunnel-api/src/main/java/org/apache/seatunnel/api/table/catalog/SeaTunnelDataTypeConvertorUtil.java

```diff
@@ -230,6 +230,9 @@ private static SeaTunnelDataType<?> parseArrayType(String field, String columnSt
                 return ArrayType.FLOAT_ARRAY_TYPE;
             case DOUBLE:
                 return ArrayType.DOUBLE_ARRAY_TYPE;
+            case MAP:
+                return new ArrayType<>(
+                        MapType.class, new MapType<>(BasicType.STRING_TYPE, BasicType.STRING_TYPE));
```

Comment:
Not all array map type is `Array<Map<String, String>>`. The Map key/value can be other type, for example `Array<Map<String, Int>>`